22_pipeslab/pipeslab.txt

```diff
@@ -1,21 +1,23 @@
 For each of the following, type a single command line that performs:
 
   1. Print the number of lines from "ls -l" which represent a path that has execute permission to their owner.
-  Answer: 
+  Answer: ls -l | cut -c4 | grep x | wc | tr -s ' ' | cut -d" " -f2
```

Comment:
you can pass flags to ```wc``` to control it's output. for example with the ```-l``` flag it will only print line count, then we can just:
```ls -l | cut -c4 | grep x | wc -l``` :)